core/client/fs/src/main/java/alluxio/client/file/options/InStreamOptions.java

```diff
@@ -55,11 +57,24 @@ public InStreamOptions(URIStatus status) {
    * @param options OpenFile options
    */
   public InStreamOptions(URIStatus status, OpenFilePOptions options) {
+    // Create OpenOptions builder with default options.
```

Comment:
Is this missing the `USER_FILE_WRITE_LOCATION_POLICY` default?

Reply:
OpenPOptions may have not contain a value for it due to not requiring client defaults.

InStreamOptions and OutStreamOptions are not compatible with this behavior change. These two actually requires the defaults so they read the Configuration object to get the defaults and then merge these with the given options object in case some have been set explicitly.